advocacy_docs/kubernetes/cloud_native_postgresql/interactive/index.mdx

```diff
@@ -0,0 +1,7 @@
+---
+title: 'Cloud Native PostgreSQL Interactive Demonstrations'
+navTitle: 'Interactive Demos'
+product: 'Cloud Native Operator'
+indexCards: full
+showInteractiveBadge: true
```

Comment:
When running the `scripts/source/source_cloud_native_operator.py` script, the `showInteractiveBadge` front matter directive is not generated.

Same thing with `advocacy_docs/kubernetes/cloud_native_postgresql/interactive/installation_and_deployment.mdx`.

Reply:
Yep - these should have been modified in the merge_source directory in order for the changes to be preserved.